dotCMS/src/main/webapp/html/portlet/ext/templates/design/design_templates.jsp

```diff
@@ -394,312 +394,290 @@
 	
 			<%-- Start Template Controls --%>
 	
-			<div class="buttonRow-left lineRight" id="editContentletButtonRow" style="height: 100%; min-height: 617px;position:absolute;top:0px;left:0px;">
-			<%
-				if(null!=parameters) { // retrieve the parameters for auto-populate the fields
-			%>
-				<div class="gradient2">
-	
-	
-	
-					<div class="fieldWrapperSide">
-						<div class="leftProperties">
-							<dl>
+			<div class="buttonRow-left lineRight portlet-sidebar-wrapper" id="editContentletButtonRow" style="height: 100%; min-height: 617px;position:absolute;top:0px;left:0px;">

```

Comment:
Also please create a portlet scss file for this file so you can do something like: ```.design-template__sidebar-wrapper``` and you can put there all the inline styles in this line (and maybe some others)